### ISSUE TITLE
Product list component stying

### ISSUE THREAD

**Issue by paddyow** [Kademi/kademi-public#13]
Looking at leading ecom sites, the product list components are clean and simple and don't use strong colours.

My suggestions are:

1) We have two options with panel and without panel (as we do now)
1b) we remove the option to choose brand colour in the configuration 
2) With panel we use a theme grey BG colour NOT brand colour as colour detracts from the product image which is hero
3) The title uses a strong P tag
4) Description uses P tag
5) Price uses P tag (but unsure about this just now)

![image](https://user-images.githubusercontent.com/2201340/91368876-8b92c900-e84d-11ea-9e71-b522c1e472ef.png)

**Comment by anhnt** [comment]
I agree with that. Price should not be outstanding, but the image must be so that user would buy the product
